Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -214,6 +262,14 @@ TrackFindingAlgorithm::TrackFindingAlgorithm(Config config,
         "required for staying on seed.");
   }
 
+  if (m_cfg.trackSelectorCfg.has_value()) {
+    m_trackSelector = std::visit(
```

Comment:
Sorry, I just spotted this one on (what I hoped was) my last review.

I guess it is neater to move this out of the constructor, but fear this probably means at least one extra copy of the whole `TrackSelector`. But maybe that was already the case before: I hear RVO doesn't work with `std::optional`.

Would it be better to `emplace` into `std::optional<Acts::TrackSelector>`?

Reply:
It might be an additional copy. I wonder if the optimizer can see through this. Also since the constructed `TrackSelector` is an rvalue.

Anyways I don't think the constructor is performance critical since we only call it a single time when the sequence is created. I would buy the copy for the improved readability.